OutOfSchool/OutOfSchool.DataAccess/Repository/CodeficatorRepository.cs

```diff
@@ -46,9 +50,9 @@ public async Task<List<CodeficatorAddressDto>> GetFullAddressesByPartOfName(stri
                         && (string.IsNullOrEmpty(namePart) && (parentId == 0) &&
                             !(categories.Contains(CodeficatorCategory.SpecialStatusCity.Name) || categories.Contains(CodeficatorCategory.Region.Name))
                        ? EF.Property<bool>(e, "IsTop")
-                       : ((e.Name.StartsWith(namePart) &&
+                       : ((e.Name.Replace('’'.ToString(), string.Empty).StartsWith(namePart) &&
```

Comment:
(Code style) Just reminding you, that there is shorter variant for `'’'.ToString()` - this one `"’"`. Could be changed here and below.

Reply:
Fixed. Thank you!